src/miniscript/types/extra_props.rs

```diff
@@ -217,6 +235,8 @@ impl Property for ExtData {
             ops_count_static: 4,
             ops_count_sat: Some(4),
             ops_count_nsat: None,
+            stack_elem_count_sat: Some(1),
+            stack_elem_count_dissat: None,
```

Comment:
If we allow malleable dissat, this should be Some(1). Same thing for rest of the hashes

Reply:
Note that hash fragment is 

`SIZE <32> EQUALVERIFY SHA256 <h> EQUAL`